doc/nrf/ug_partition_manager.rst

```diff
@@ -0,0 +1,189 @@
+.. _ug_pm:
+
+Partition Manager
```

Comment:
I it possible to change this name to something more specific for the build system - on the firs sight `Partition manager` sound like software component used in fimware for software domain isolation. `Partition build manager`

I know that it is late comment - so can be ignored

Reply:
or `Partition images build manager`

Reply:
@carlescufi, @pizi-nordic Have you any thought on that?

Reply:
Image partitioner?

Reply:
Note that this component will tackle sharing of RAM resources as well soon.

Reply:
so the word `partition` fit for both RAM and ROM partition.

Reply:
That was my thought at least. I can not foresee any other role that this component will handle in a multi image context.